### PR TITLE
Add Css rules for custom properties fields

### PR DESCRIPTION
**Pull request by meven**
Signed-off-by: Méven Car <meven.car@collabora.com>
Change-Id: I4b57a1c04dc67620f4ee010e17fff4f9e677d0de (cherry picked from commit 8b2f8916b3490d273c3690125ad18aeb46d69924)

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by pedropintosilva — COMMENTED** [review]
Doesn't this need the `Allow Document properties tab custom properties for LOK` in the core 23 ?

**Comment by meven** [comment]
> Doesn't this need the `Allow Document properties tab custom properties for LOK` in the core 23 ?

It does.
But this is not a hard-dependency (it can get in without any issue).

File: browser/css/jsdialogs.css [hunk 1]
```diff
@@ -1913,3 +1913,13 @@ kbd,
 .jsdialog-container [id^='info-modal-label'] {
 	display: flex;
 }
+
+/* Custom props */
+#customprops .jsdialog .ui-grid .menubutton {
```
Comment (eszkadev) [inline browser/css/jsdialogs.css:1918]: Master version is different (and better in my opinion, prevents applying to something different):
https://github.com/CollaboraOnline/online/blob/913f684e3023e1dc43a6c0ef99c8ff1428c74a88/browser/css/jsdialogs.css#L2077
`#customprops` -> `#DocumentPropertiesDialog #customprops`

**Review by eszkadev — CHANGES_REQUESTED** [review]
sync with master

**Review by eszkadev — APPROVED** [review]

**Pull request merged**